c7n/tags.py

```diff
@@ -80,8 +80,16 @@ def universal_augment(self, resources):
     if not resources:
         return resources
 
+    # For global resources, tags don't populate in the get_resources call
+    # unless the call is being made to us-east-1
+    if hasattr(self.resource_type, 'global_resource') \
```

Comment:
use getattr with a default to simplify the conditional, could even one line the whole thing with a ternary operator / conditional expression, ie.

```
region = getattr(self.resource_type, 'global_resource', None) and 'us-east-1' or self.region
```